src/components/UserInfo.js

```diff
@@ -14,6 +19,10 @@ class UserInfo {
     this._title.textContent = title;
     this._description.textContent = description;
   }
+
+  setAvatar(avatar) {
```

Comment:
It's great to have a separate setter for the avatar, since it's controlled by a different form